docs/en/02-how-to-run/profile_model.md

```diff
@@ -55,7 +61,8 @@ python tools/test.py \
     {MMCLS_DIR}/configs/resnet/resnet50_b32x8_imagenet.py \
     --model model.onnx \
     --out out.pkl \
-    --device cuda:0
+    --device cuda:0 \
```

Comment:
`--device cuda:0` -> `--device cpu`, since it's onnxruntime

Reply:
done.